src/main/kotlin/com/redhat/devtools/intellij/kubernetes/model/client/ClientConfig.kt

```diff
@@ -54,15 +54,14 @@ open class ClientConfig(private val client: Client, private val executor: Execut
 					return@supplyAsync false
 				}
 				val fromFile = kubeConfig.load() ?: return@supplyAsync false
-				val currentContextInFile = KubeConfigUtils.getCurrentContext(fromFile)
```

Comment:
This was the error: storing the current context before the new context is set and then setting the namespace on it:
`#setCurrentContext` would change the current context and `#setCurrentNamespace` would then change the namespace on the current context that was "current" before changing to a new one.